app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentConvertDialog.java

```diff
@@ -0,0 +1,39 @@
+package com.mithridat.nonoconverter.ui.editimage;
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+
+import com.mithridat.nonoconverter.R;
+
+import androidx.fragment.app.DialogFragment;
+import androidx.fragment.app.FragmentActivity;
+
+public class FragmentConvertDialog extends DialogFragment{
+
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+        FragmentActivity fragmentActivity = getActivity();
+        if (fragmentActivity == null) return null;
+        androidx.appcompat.app.AlertDialog.Builder builder =
+                new androidx.appcompat.app.AlertDialog.Builder(fragmentActivity)
+                .setTitle(R.string.title_dialog_convert)
+                .setIcon(R.drawable.ic_info)
+                .setMessage(R.string.msg_convert_ok)
+                .setPositiveButton(R.string.action_ok, listenerOk);
+        return builder.create();
```

Comment:
It can be replaced by
```java
return new AlertDialog.Builder(getActivity())
                .setTitle(R.string.title_dialog_convert)
                .setIcon(R.drawable.ic_info)
                .setMessage(R.string.msg_convert_ok)
                .setPositiveButton(R.string.action_ok, listenerOk)
                .create();
```